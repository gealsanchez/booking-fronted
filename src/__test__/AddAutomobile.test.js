import React from 'react';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import AddAutomobile from '../components/AddAutomobile';

it('Add automobile component should render correctly', () => {
  const result = render(
    <React.StrictMode>
      <Router>
        <Provider store={store}>
          <AddAutomobile />
        </Provider>
      </Router>
    </React.StrictMode>,
  );
  expect(result).toMatchSnapshot();
});
