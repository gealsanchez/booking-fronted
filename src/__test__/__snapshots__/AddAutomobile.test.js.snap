// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add automobile component should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="add-automobile-form"
      >
        <p />
        <p
          class="add-automobile-title"
        >
          Add an automobile
        </p>
        <div
          class="inputs"
        >
          <input
            class="model-automobile-input"
            placeholder="Model"
            type="text"
            value=""
          />
          <input
            class="year-input"
            placeholder="prd-year"
            type="text"
            value=""
          />
          <input
            class="location-input"
            placeholder="City"
            type="text"
            value=""
          />
          <input
            class="rate-input"
            placeholder="Charging rate $"
            type="text"
            value=""
          />
          <input
            class="photo-input"
            placeholder="Photo Link"
            type="text"
            value=""
          />
          <button
            class="add-automobile-button"
            type="button"
          >
            Add automobile
          </button>
          <a
            class="link back-automobiles"
            href="/main"
          >
            Back to automobiles
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="add-automobile-form"
    >
      <p />
      <p
        class="add-automobile-title"
      >
        Add an automobile
      </p>
      <div
        class="inputs"
      >
        <input
          class="model-automobile-input"
          placeholder="Model"
          type="text"
          value=""
        />
        <input
          class="year-input"
          placeholder="prd-year"
          type="text"
          value=""
        />
        <input
          class="location-input"
          placeholder="City"
          type="text"
          value=""
        />
        <input
          class="rate-input"
          placeholder="Charging rate $"
          type="text"
          value=""
        />
        <input
          class="photo-input"
          placeholder="Photo Link"
          type="text"
          value=""
        />
        <button
          class="add-automobile-button"
          type="button"
        >
          Add automobile
        </button>
        <a
          class="link back-automobiles"
          href="/main"
        >
          Back to automobiles
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
